From c74079093e9543f27501686994015b2084206bcd Mon Sep 17 00:00:00 2001
From: Arzhan Kinzhalin <arzhan.i.kinzhalin@intel.com>
Date: Mon, 25 Sep 2017 17:41:48 +0000
Subject: [PATCH] Use p11-trust instead of trust.

Clear Linux provides a stub instead of 'trust' to avoid using it for
trust store management. The native p11-kit 'trust' command is provided
as 'p11-trust' instead.
---
 p11-kit/p11-kit.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/p11-kit/p11-kit.c b/p11-kit/p11-kit.c
index 7a90046..fabb230 100644
--- a/p11-kit/p11-kit.c
+++ b/p11-kit/p11-kit.c
@@ -76,7 +76,7 @@ p11_kit_trust (int argc,
 	args = calloc (argc + 2, sizeof (char *));
 	return_val_if_fail (args != NULL, 1);
 
-	args[0] = BINDIR "/trust";
+	args[0] = BINDIR "/p11-trust";
 	memcpy (args + 1, argv, sizeof (char *) * argc);
 	args[argc + 1] = NULL;
 
-- 
2.13.2

